main.go

```diff
@@ -44,29 +42,20 @@ func main() {
 	}
 
 	ebpfEnabled, _ := strconv.ParseBool(os.Getenv("EBPF_ENABLED"))
-	metricsEnabled, _ := strconv.ParseBool(os.Getenv("METRICS_ENABLED"))
 
-	// datastore backend
-	dsBackend := datastore.NewBackendDS(ctx, config.BackendConfig{
-		Host:                  os.Getenv("BACKEND_HOST"),
-		Port:                  os.Getenv("BACKEND_PORT"),
-		MetricsExport:         metricsEnabled,
-		MetricsExportInterval: 10,
-	})
-	go dsBackend.SendHealthCheck(ebpfEnabled, metricsEnabled)
+	// start Prometheus exporter
+	exporter := datastore.NewPrometheusExporter(ctx)
 
 	// deploy ebpf programs
 	var ec *ebpf.EbpfCollector
 	if ebpfEnabled {
 		ec = ebpf.NewEbpfCollector(ctx)
 		go ec.Deploy()
 
-		a := aggregator.NewAggregator(kubeEvents, nil, ec.EbpfEvents(), dsBackend)
+		a := aggregator.NewAggregator(kubeEvents, nil, ec.EbpfEvents(), exporter)
 		a.Run()
 	}
 
-	go http.ListenAndServe(":8181", nil)
```

Comment:
I have no clue what this was for, but it doesn't seem to be necessary.

Reply:
> Container port for debugging and profiling Alaz

as per https://github.com/ddosify/ddosify-helm-charts#alaz-parameters